### PR TITLE
(feat)graphQL - add mutation for creating cohort

### PR DESCRIPTION
**Pull request by dmukherjee**
This PR addresses:

- Adding cohort through graphQL
- Refactor helper functions for rest api to use camel case variable names

### CODE REVIEW

**Review by michellelockett — APPROVED** [review]
This looks great!

**Comment by michellelockett** [comment]
This looks great, I think we are all on the same page with snake_case in db and camelCase everywhere else - the only thing we have left to sort out is whether we use cohortId, studentId, or id for both.  @magee would love your thoughts.

File: server/schema/schema.js [hunk 1]
```diff
@@ -216,6 +217,29 @@ const RootQuery = new GraphQLObjectType({
...
+      args: {
+        name: { type: GraphQLString },
+        phase: { type: GraphQLString },
+      },
+      resolve(parent, args) {
+        let { name, phase } = args;
+        name = name.toLowerCase();
+        return cohorts.addCohort({ name, phase })
+          .then((result) => {
+            return { id: result.cohort_id, name: result.cohort_name, phase: result.phase }
```
Comment (michellelockett) [inline server/schema/schema.js]: We need to decide whether to do the mapping of snake_case to camelCase in the graphQL layer or the db models.  I think a case could be made for either, so long as the client requests include the simplified camelCase.  @magee preferences?

Comment (dmukherjee) [reply server/schema/schema.js]: @michellelockett I pushed in a change to do the mapping in db model. Please take a look.

**Pull request merged**